src/app/features/genes/components/gene-hero/gene-hero.component.html

```diff
@@ -14,17 +14,31 @@ <h2 class="gene-hero-name">
       <p class="gene-hero-provider">
         {{ getSummary() }}
       </p>
-      <ng-container *ngIf="gene.bio_domains || getAlias()">
+      <ng-container *ngIf="gene.bio_domains || getAlias() || getEnsemblUrl() !== ''">
         <hr />
         <div *ngIf="gene.bio_domains" class="gene-hero-biodomains">
           <h4 class="gene-hero-biodomains-heading">Biological Domains</h4>
           <p>
             {{ getBiodomains() }}
           </p>
         </div>
-        <div *ngIf="getAlias()" class="gene-hero-aliases">
+        <div *ngIf="getAlias() || getEnsemblUrl() !== ''" class="gene-hero-aliases">
           <h4 class="gene-hero-aliases-heading">Also known as</h4>
-          <p *ngIf="gene.hgnc_symbol" class="mb-0">{{ gene.ensembl_gene_id }}</p>
+          <ng-container *ngIf="getEnsemblUrl() !== ''">
+            <p>
+              <a [href]="getEnsemblUrl()" target="_blank">{{ gene.ensembl_gene_id }}</a>
+              <span *ngIf="gene.ensembl_info[0].ensembl_release">
+                (Ensembl Release {{ gene.ensembl_info[0].ensembl_release }})
+              </span>
+            </p>
+          </ng-container>
+          <ng-container *ngIf="getEnsemblUrl() === ''">
+            <span *ngIf="getEnsemblUrl() === ''">{{ gene.ensembl_gene_id }}</span>
+          </ng-container>
+          <p *ngIf="gene.ensembl_info[0].ensembl_possible_replacements.length > 0">
+            <a [href]="getPossibleReplacementsURL()">Possible replacement value<span *ngIf="gene.ensembl_info[0].ensembl_possible_replacements.length > 1">s</span></a>:
+            {{ gene.ensembl_info[0].ensembl_possible_replacements.join(', ')}}
+          </p>
           <p>
             {{ getAlias() }}
           </p>
```

Comment:
Should a conditional be added to the `getAlias()` paragraph, since the parent div is no longer conditionally showed if the `getAlias()` exists?

Reply:
correct.  I fixed the parent div and added the condition